lib/websocket_server.rb

```diff
@@ -24,7 +26,10 @@ def initialize
         reads.each do |read|
           begin
             @pairing[read].proxy.transmit(writes, @pairing[read].is_ws)
-          rescue
+          rescue IOError
+            cleanup(errors)
+          rescue => ex
+            @logger.error(@pairing[read].proxy.crash_report(ex))
```

Comment:
Is it possible for `@pairing[read].proxy` to be nil here?  We're in an exception handler... I don't know.